rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] According to the project requirements, since book and person can also be modified, it should be a getter and setter instance variable, so it would be better made accessible with `attr_accessor`